packages/smart-forms-renderer/src/stories/itemTypes/OpenChoice.stories.tsx

```diff
@@ -37,30 +36,179 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 // More on writing stories with args: https://storybook.js.org/docs/react/writing-stories/args
+const clinicCoding = {
+  system: 'http://snomed.info/sct',
+  code: '257585005',
+  display: 'Clinic'
+}
+const targetText = 'Pharmacy'
+
+const qOpenChoiceAnswerOptionBasic = questionnaireFactory([{
+  linkId: 'health-check-location',
+  text: 'Location of health check',
+  type: 'open-choice',
+  extension: [
+    itemControlExtFactory('radio-button'),
+    {
```

Comment:
to be consistent, define factory for this extension too. our goal it finally to have all extensions with factories